packages/db/test/integrations.test.js

```diff
@@ -10,10 +10,10 @@ describe('astro:db with integrations', () => {
 		});
 	});
 
-	// Note(bholmesdev): Use in-memory db to avoid
-	// Multiple dev servers trying to unlink and remount
-	// the same database file.
-	process.env.TEST_IN_MEMORY_DB = 'true';
+	// Note (@bholmesdev) generate a random database id on startup.
+	// Ensures database connections don't conflict
+	// when multiple dev servers are run in parallel on the same project.
+	process.env.ASTRO_TEST_RANDOM_DB_ID = 'true';
```

Comment:
Where was this changed?

Reply:
Ah, we added the check here when we generate the database URL: https://github.com/withastro/astro/blob/main/packages/db/src/core/consts.ts#L16

Is this what you are asking?

Reply:
Update: I just removed the need for this code 😅 No need for the comment!